source/Server.Contracts/Accounts/IExpandVariableForAccountDetails.cs

```diff
@@ -0,0 +1,13 @@
+using System.Collections.Generic;
+using Sashimi.Server.Contracts.Variables;
+
+namespace Sashimi.Server.Contracts.Accounts
+{
+    public interface IExpandVariableForAccountDetails
```

Comment:
Some of these signatures are different from Server, do the same refactoring on the Server side before deleting the Server version